usersimcrs/user_modeling/preference_model.py

```diff
@@ -143,7 +143,8 @@ def get_item_preference(self, item_id: str) -> float:
         return preference
 
     def _check_slot_exists(self, slot: str) -> None:
-        """Checks if slot exists in the domain and throws an exception if not."""
```

Comment:
The original docstring was within the 80chars limit, no?

Re: your earlier comment, I still think this method should not have a return value, but `_assert_slot_exists()` might be a better name then.

Reply:
The docstring was 81 characters, so I ran docformatter to fix that.

I like to new name for the method.